### ISSUE TITLE
CAPTCHA user-reported bug

### ISSUE THREAD

**Issue by AmaniKrayemRCPCH** [rcpch/rcpch-audit-engine#801]
Received the below message from a user - "I am unable to log on as the new platform  has CAPTCHA on it- 
CAPTCHA works but  One has to try 47 times or something before it allows you. I have just tried 49 times but it locks me out. 
Hence, I will be unable to log on  and take part . I am happy to help if anybody is able to log on the system for me. IT have agreed, CAPTCHA is unworkable, quite secure and very unpractical. 
It would be helpful if CAPTHCHA can be treated off from the new platform."

Unsure what the problem is, and CAPTCHA appears to be working. I have asked the user for a screenshot.

**Comment by pacharanero** [comment]
Yes, it would be helpful if we can get further details from the user of exactly what is happening when it "doesn't work". In general when anyone reports that something "doesn't work" they are **by definition** not including enough information to debug **any** problem. "Doesn't work" is the _absence_ of an expected behaviour, so we need a description of exactly what DOES happen. I agree with you Amani that a screenshot would be helpful here.

CAPTCHA was a recommendation of the Privacy Impact Assessment I think, and/or of the penetration testers. However I agree that CAPTCHAs are annoying. If we can ensure adequate security without CAPTCHA then I am in favour of switching it off. I wonder if perhaps we could use some kind of firewall filtering to exclude non-UK traffic automatically, since this is likely to filter out a lot of malicious login attempts.

It is strange that the user is somehow able to log in after a very large number of attempts, this is an odd behaviour, usually when tech works it either works all the time or fails all the time. An 'in-between' state like this is interesting. More info please.

**Comment by AmaniKrayemRCPCH** [comment]
I'm unsure about this user, as I haven't gotten a reply, but I got a similar issue over the phone. 

It turns out they hadn't actually set up their account after it was created. So their email/password was incorrect, but they were interpreting it as a CAPTCHA error. I asked and they said there wasn't an 'Invalid CAPTCHA' message. Once they reset their password, all was resolved. 

Not sure if the same applies to this user, considering they've gotten their IT team involved. User logs show that they have had a successful log in since this was raised.
